services/text_to_speech/tts-utils.js

```diff
@@ -15,26 +15,30 @@
  **/
 
 const pkg = require('../../package.json'),
-  TextToSpeechV1 = require('watson-developer-cloud/text-to-speech/v1');
+  TextToSpeechV1 = require('ibm-watson/text-to-speech/v1'),
+  { IamAuthenticator } = require('ibm-watson/auth');
 
 class TTSUtils {
   constructor() {
   }
 
   static buildStdSettings (apikey, username, password, endpoint) {
+    let authSettings  = {};
     let serviceSettings = {
       headers: {
         'User-Agent': pkg.name + '-' + pkg.version
       }
     };
 
     if (apikey) {
-      serviceSettings.iam_apikey = apikey;
+      authSettings.apikey = apikey;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 spaces but found 6.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4764899)